src/components/Card/DBCard.tsx

```diff
@@ -1,40 +1,82 @@
+'use client';
+
 import { DBCardArrowIcon } from '@/public/index';
+import { useDBCardStore } from '@/stores/DBCardStore';
 import { format } from 'date-fns';
+import Link from 'next/link';
 import { twMerge } from 'tailwind-merge';
 
 type DBCardProps = {
+  id: number;
   title: string;
   date: Date;
   backgroundId: number;
 };
 
-export default function DBCard({ title, date, backgroundId }: DBCardProps) {
-  const backgroundById = (id: number) => {
-    if (id === 1) {
+export default function DBCard({ id, title, date, backgroundId }: DBCardProps) {
+  const { isHoverFirstCard, setIsHoverFirstCard } = useDBCardStore();
+  const conditionOfFirstDBCard = backgroundId === 1 && isHoverFirstCard;
+  const backgroundById = (bgId: number) => {
+    if (bgId === 1) {
       return 'bg-[linear-gradient(360deg,rgba(0,0,0,0.7),rgba(255,255,255,0)),url(/images/DBCardDummyImg1.png)]';
-    } else if (id === 2) {
+    } else if (bgId === 2) {
       return 'bg-[linear-gradient(360deg,rgba(0,0,0,0.7),rgba(255,255,255,0)),url(/images/DBCardDummyImg2.png)]';
     } else {
       return 'bg-[linear-gradient(360deg,rgba(0,0,0,0.7),rgba(255,255,255,0)),url(/images/DBCardDummyImg3.png)]';
     }
   };
+  const onMouseOver2And3 = () => {
+    if (backgroundId !== 1) {
+      setIsHoverFirstCard(false);
+    }
+  };
+  const onMouseOut2And3 = () => {
+    if (backgroundId !== 1) {
+      setIsHoverFirstCard(true);
+    }
+  };
```

Comment:
```suggestion
const handleHover = (isHovering: boolean) => {
  if (backgroundId !== 1) {
    setIsHoverFirstCard(isHovering);
  }
};
```
P4: 어딘가 중복같이 느껴져 줄이고 싶은 욕심이 나용,,ㅎ